File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        v=[]
        
        for i in range(1,n//2+1):
            v.append(i)
            v.append(-i)
        
        if n&1:
            v.append(0)
        return v
        

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> v(n);
        int j{n-1};
        
        for(int i=1;i<=n/2;++i,--j){
          v[i-1]=i;
          v[j]=0-i;
        }
        if(n&1) v[j]=0;
        
        return v;
    }
};